blockchain/sync_manager/src/errors.rs

```diff
@@ -37,20 +38,14 @@ impl fmt::Display for Error {
         match self {
             Error::NoBlocks => write!(f, "No blocks for tipset"),
             Error::InvalidRoots => write!(f, "Invalid root detected"),
-            Error::Blockchain(msg) => write!(
-                f,
-                "Error originating from construction of blockchain structures: {}",
-                msg
-            ),
-            Error::KeyValueStore(msg) => {
-                write!(f, "Error originating from Key-Value store: {}", msg)
-            }
-            Error::Encoding(msg) => write!(f, "Error originating from Encoding type: {}", msg),
-            Error::InvalidCid(msg) => write!(f, "Error originating from CID construction: {}", msg),
-            Error::Store(msg) => write!(f, "Error originating from ChainStore: {}", msg),
-            Error::AMT(msg) => write!(f, "Error originating from the AMT: {}", msg),
-            Error::State(msg) => write!(f, "Error originating from the State: {}", msg),
-            Error::Validation(msg) => write!(f, "Error validating data: {}", msg),
+            Error::Blockchain(msg) => write!(f, "{}", msg),
+            Error::KeyValueStore(msg) => write!(f, "{}", msg),
+            Error::Encoding(msg) => write!(f, "{}", msg),
+            Error::InvalidCid(msg) => write!(f, "{}", msg),
+            Error::Store(msg) => write!(f, "{}", msg),
+            Error::AMT(msg) => write!(f, "{}", msg),
+            Error::State(msg) => write!(f, "{}", msg),
+            Error::Validation(msg) => write!(f, "{}", msg),
```

Comment:
Shouldn't the error type be specified as part of the returned message? `msg` may be ambiguous

Reply:
Discussed error messages with @austinabell last week and decided to remove the extra message as it looked quite messy with the included `msg`. Also, the error variant type would indicate the error type. Nonetheless, I am in favor of determining a standard for our error.rs files moving forward.

Reply:
Ya the reason another message isn't appended to these are they are essentially forwarded error messages from another type. Can rework how we handle errors or in this case have the error it is forwarding be included as the variant type instead of `String`

Reply:
#216